craft_application/remote/git.py

```diff
@@ -221,7 +221,7 @@ def _init_repo(self) -> None:
                 f"Could not initialize a git repository in {str(self.path)!r}."
             ) from error
 
-    def push_url(  # pylint: disable=too-many-branches
+    def push_url(  # noqa: PLR0912 (too many branches)
```

Comment:
```suggestion
    def push_url(  # noqa: PLR0912 (too-many-branches)
```
I saw that usually we use dashed version of the rule